src/config.c

```diff
@@ -115,7 +115,7 @@ configEnum oom_score_adj_enum[] = {
 
 configEnum acl_pubsub_default_enum[] = {
     {"allchannels", USER_FLAG_ALLCHANNELS},
-    {"resetchannels", 0},
```

Comment:
So if someone already has this in their config file, I don't think we can just break them. I would prefer we say nochannels is just a synonym for resetchannels. We can remove it in 7.0 perhaps.

Reply:
Yeah, I was also thinking the same, christoph@ had the same concern. Will keep both of them and treat them equally. I hope that works.